### PR TITLE
Add 2023 H2  Roadmap_zh

### PR DESCRIPTION
**Pull request by zhiyingfang2022**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it:**
Hi ~Add 2023 H2  Roadmap_zh to provide direction for subsequent version planning.

cc @fisherxu @Shelley-BaoYue

### CODE REVIEW

File: docs/roadmap_zh.md [hunk 1]
```diff
@@ -59,4 +59,98 @@
...
+
+## 2023 H2
+
+### SIG Node
+
+- 支持在Windows上运行边缘节点。
+- 对边缘节点进行功能增强，如支持静态Pod、事件报告和可配置的应用迁移策略。
+- 支持在RTOS系统上运行边缘节点。
+- 对节点上的设备插件进行改进，如支持多个虚拟GPU。
+- 支持无服务器计算。
```
Comment (fisherxu) [inline docs/roadmap_zh.md:72]:
```suggestion
- 支持边缘Serverless。
```
用英文就行

File: docs/roadmap_zh.md [hunk 2]
```diff
@@ -59,4 +59,98 @@
...
+- 远程升级边缘节点特性 GA。
+- 优化节点组功能，如支持更多差异化配置参数。
+
+### SIG Device-IOT
+
+- 支持DMI数据平面（H1已完成设计）。
+- 基于DMI的边缘设备在多节点间的迁移解决方案。
+- 支持DMI的映射器框架。
```
Comment (fisherxu) [inline docs/roadmap_zh.md:80]:
```suggestion
- 更新基于最新DMI框架的内置Mapper。
```

File: docs/roadmap_zh.md [hunk 3]
```diff
@@ -59,4 +59,98 @@
...
+- 研究支持多语言映射器。
+- 与时序数据库和其他数据库集成。
+- 重构设备和设备模型CRD。
```
Comment (fisherxu) [inline docs/roadmap_zh.md:83]:
```suggestion
- 重构DeviceInstance和Device Model API。
```

File: docs/roadmap_zh.md [hunk 4]
```diff
@@ -59,4 +59,98 @@
...
+- 提高自定义消息传输的可靠性。
+
+### SIG Security
+
+- SLSA / CodeQL（仍有一些溯源工作需要达到SLSA L4）。
+- Spiffe研究。
+- 支持具有多种加密算法的证书，并提供接口能力。
```
Comment (fisherxu) [inline docs/roadmap_zh.md:90]:
```suggestion
- 提供通用接口，支持多种加密算法的证书。
```

**Review by fisherxu — COMMENTED** [review]
可以删了每句话后的句号

File: docs/roadmap_zh.md [hunk 5]
```diff
@@ -59,4 +59,98 @@
...
+### Docs
+
+- 优化网站文档，包括目录重组和增强全面性。
+- 支持更新文档版本。
+- 完善官方网站文档，包括DMI开发者指南、监控操作指南等。
+- 在网站案例中心发布案例。
+
+### UI
+
+- 仪表板发布迭代版本支持。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:137]:
```suggestion
- Dashboard版本迭代。
```

File: docs/roadmap_zh.md [hunk 6]
```diff
@@ -59,4 +59,98 @@
...
+- 在网站上新增案例中心。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:138]:
```suggestion
- 在官网新增案例中心
```

File: docs/roadmap_zh.md [hunk 7]
```diff
@@ -59,4 +59,98 @@
...
+
+### SIG Cluster-Lifecycle
+
+- 支持Windows安装和部署。
+- 边缘应用程序的预下载镜像。
+- 消息路由高可用性（HA）支持。
+
+### Docs
+
+- 优化网站文档，包括目录重组和增强全面性。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:130]:
```suggestion
- 官网文档优化，包括目录重构以及完备性提升
```

File: docs/roadmap_zh.md [hunk 8]
```diff
@@ -59,4 +59,98 @@
...
+- 改善基于场景的e2e测试用例覆盖范围。
+- 集成测试。
+- 运行时和K8s版本兼容性测试。
+- Keadm跨版本兼容性测试。
+- 云-边缘跨版本兼容性测试。
+
+### SIG Networking
+
+- 节点离线优化。
+- 当节点离线时，其他节点会收到更新并从终结点中删除相应的后端。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:115]: 这里格式不太对，节点离线优化和大规模优化应该是两个次级标题

File: docs/roadmap_zh.md [hunk 9]
```diff
@@ -59,4 +59,98 @@
...
+- 对边缘节点进行功能增强，如支持静态Pod、事件报告和可配置的应用迁移策略。
+- 支持在RTOS系统上运行边缘节点。
+- 对节点上的设备插件进行改进，如支持多个虚拟GPU。
+- 支持无服务器计算。
+- 远程升级边缘节点特性 GA。
+- 优化节点组功能，如支持更多差异化配置参数。
+
+### SIG Device-IOT
+
+- 支持DMI数据平面（H1已完成设计）。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:78]:
```suggestion
- 支持DMI数据面（H1已完成设计）。
```

File: docs/roadmap_zh.md (hunk 2 above)
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:80]: Mapper就用英文，不用翻译

File: docs/roadmap_zh.md [hunk 10]
```diff
@@ -59,4 +59,98 @@
...
+- 基于DMI的边缘设备在多节点间的迁移解决方案。
+- 支持DMI的映射器框架。
+- 研究支持多语言映射器。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:81]: 多语言Mapper支持调研

File: docs/roadmap_zh.md [hunk 11]
```diff
@@ -59,4 +59,98 @@
...
+- 与时序数据库和其他数据库集成。
+- 重构设备和设备模型CRD。
+- 提高自定义消息传输的可靠性。
+
+### SIG Security
+
+- SLSA / CodeQL（仍有一些溯源工作需要达到SLSA L4）。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:88]:
```suggestion
- SLSA / CodeQL（距离达到SLSA L4还有一些来源证据链的工作）。
```

File: docs/roadmap_zh.md [hunk 12]
```diff
@@ -59,4 +59,98 @@
...
+- Spiffe研究。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:89]: 研究翻译成调研比较好

File: docs/roadmap_zh.md [hunk 13]
```diff
@@ -59,4 +59,98 @@
...
+
+### SIG Scalability
+
+- 集成EdgeMesh的可扩展性和性能测试。
+- 针对IoT设备场景的可扩展性和性能测试。
+
+### Stability
+
+- CloudCore的稳定性维护，包括稳定性测试和问题解决。
+- dgeMesh的稳定性。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:100]:
```suggestion
- EdgeMesh的稳定性工作
```

File: docs/roadmap_zh.md [hunk 14]
```diff
@@ -59,4 +59,98 @@
...
+- 提高云端与边缘协作的可靠性，如改进Edge Kube-API接口和日志/执行功能的稳定性。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:101]:
```suggestion
- 云边协同的可靠性增强，如提升Edge Kube-API接口和logs/exec能力的稳定性。
```

File: docs/roadmap_zh.md [hunk 15]
```diff
@@ -59,4 +59,98 @@
...
+
+### SIG Testing
+
+- 增加单元测试覆盖率。
+- 改善基于场景的e2e测试用例覆盖范围。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:106]:
```suggestion
- E2e测试用例覆盖率提升（按场景覆盖）
```

File: docs/roadmap_zh.md [hunk 16]
```diff
@@ -59,4 +59,98 @@
...
+- 集成测试。
+- 运行时和K8s版本兼容性测试。
+- Keadm跨版本兼容性测试。
+- 云-边缘跨版本兼容性测试。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:110]: Keadm版本交叉兼容性测试，云边版本交叉兼容性测试

File: docs/roadmap_zh.md [hunk 17]
```diff
@@ -59,4 +59,98 @@
...
+
+### SIG Robotics
+
+- 添加RoboDev仓库：为开发人员更轻松地构建机器人应用程序。
+- 添加RTF（即开即飞）Robotics E2E解决方案：远程操作和机器人驾驶员。
+
+### Experience
+
+- 示例库增强
+- Killer-Coda上线
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:156]:
```suggestion
- 上线到Killer-Coda
```

File: docs/roadmap_zh.md [hunk 18]
```diff
@@ -59,4 +59,98 @@
...
+- 支持更新文档版本。
+- 完善官方网站文档，包括DMI开发者指南、监控操作指南等。
+- 在网站案例中心发布案例。
+
+### UI
+
+- 仪表板发布迭代版本支持。
+- 在网站上新增案例中心。
+- 在网站上新增招聘中心。
+- 支持网站的版本控制。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:140]:
```suggestion
- 支持官网的版本管理
```

File: docs/roadmap_zh.md [hunk 19]
```diff
@@ -59,4 +59,98 @@
...
+- 支持Windows安装和部署。
+- 边缘应用程序的预下载镜像。
+- 消息路由高可用性（HA）支持。
+
+### Docs
+
+- 优化网站文档，包括目录重组和增强全面性。
+- 支持更新文档版本。
+- 完善官方网站文档，包括DMI开发者指南、监控操作指南等。
+- 在网站案例中心发布案例。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:133]:
```suggestion
- 发布案例到官网案例中心
```

File: docs/roadmap_zh.md [hunk 20]
```diff
@@ -59,4 +59,98 @@
...
+
+- 支持Windows安装和部署。
+- 边缘应用程序的预下载镜像。
+- 消息路由高可用性（HA）支持。
+
+### Docs
+
+- 优化网站文档，包括目录重组和增强全面性。
+- 支持更新文档版本。
+- 完善官方网站文档，包括DMI开发者指南、监控操作指南等。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:132]:
```suggestion
- 完善官方网站文档，包括DMI开发者指南、对接监控操作指南等。
```

File: docs/roadmap_zh.md [hunk 21]
```diff
@@ -59,4 +59,98 @@
...
+- 大规模优化。
+- 在大规模部署中，边缘kube apiserver的负载很高。考虑使用IPVS（IP虚拟服务器）技术来有效处理请求。
+- 服务数量的增加显著增加了节点上的iptables规则数量。容器网络支持CNI特性。
+- 性能优化：基于eBPF（扩展伯克利数据包过滤器）的内核级流量转发。
+- 分布式消息系统。
+
+### SIG Cluster-Lifecycle
+
+- 支持Windows安装和部署。
+- 边缘应用程序的预下载镜像。
```
Comment (Shelley-BaoYue) [inline docs/roadmap_zh.md:125]:
```suggestion
- 支持边缘应用镜像预下载
```

File: docs/roadmap_zh.md [hunk 22]
```diff
@@ -59,4 +59,98 @@
...
+
+### SIG Scalability
+
+- 集成EdgeMesh的可扩展性和性能测试
+- 针对IoT设备场景的可扩展性和性能测试
+
+### Stability
+
+- CloudCore的稳定性维护，包括稳定性测试和问题解决
+- dgeMesh的稳定性
```
Comment (fisherxu) [inline docs/roadmap_zh.md:100]:
```suggestion
- EdgeMesh的稳定性
```

File: docs/roadmap_zh.md [hunk 23]
```diff
@@ -59,4 +59,98 @@
...
+### SIG AI
+
+- 支持边缘-云协同生命周期学习中的半自动注释
+- 支持边缘-云协同生命周期学习中的运行时未知任务处理
+- 支持边缘-云协同生命周期学习中的高级离线未知任务处理
+
+### SIG Robotics
+
+- 添加RoboDev仓库：为开发人员更轻松地构建机器人应用程序
+- 添加RTF（即开即飞）Robotics E2E解决方案：远程操作和机器人驾驶员
```
Comment (fisherxu) [inline docs/roadmap_zh.md:151]:
```suggestion
- 添加RTF（开箱即用）Robotics E2E解决方案：远程操作和机器人驾驶员
```

Comment (zhiyingfang2022) [reply docs/roadmap_zh.md]: Thank you.All modified.

File: docs/roadmap_zh.md [hunk 24]
```diff
@@ -59,4 +59,99 @@
...
+### SIG AI
+
+- 支持边缘-云协同生命周期学习中的半自动注释
+- 支持边缘-云协同生命周期学习中的运行时未知任务处理
+- 支持边缘-云协同生命周期学习中的高级离线未知任务处理
+
+### SIG Robotics
+
+- 添加RoboDev仓库：为开发人员更轻松地构建机器人应用程序
+- 添加RTF（开箱即用）Robotics E2E解决方案：远程操作和机器人驾驶员
```
Comment (JoeyHwong-gk) [inline docs/roadmap_zh.md:152]: simple suggestion：新增RTF（开箱即用）的机器人端到端解决方案：Teleoperation(遥操作) 和 RoboPilot （具身智能框架）
> 项目名使用原文即可

**Comment by JoeyHwong-gk** [comment]
BTW, It seems more suitable for submit in [Community](https://github.com/kubeedge/community) rather than here ?

**Comment by zhiyingfang2022** [comment]
> BTW, It seems more suitable for submit in [Community](https://github.com/kubeedge/community) rather than here ?

Thanks for the suggestion. After the review is completed here, it will be submitted to the community.

File: docs/roadmap_zh.md (hunk 24 above)
Comment (fisherxu) [reply docs/roadmap_zh.md:152]: PTAL, thanks @zhiyingfang2022

Comment (zhiyingfang2022) [reply docs/roadmap_zh.md]: Thanks,modified. @JoeyHwong-gk @fisherxu

**Review by fisherxu — COMMENTED** [review]
/approve
/assign @Shelley-BaoYue

**Comment by fisherxu** [comment]
@zhiyingfang2022 As @JoeyHwong-gk said, let's move the road map to community repo, and then make the link here https://github.com/kubeedge/kubeedge#roadmap  pointing to community repo, thanks :)

**Comment by zhiyingfang2022** [comment]
> @zhiyingfang2022 As @JoeyHwong-gk said, let's move the road map to community repo, and then make the link here https://github.com/kubeedge/kubeedge#roadmap pointing to community repo, thanks :)

ok

**Pull request merged**